bin/node/runtime/src/lib.rs

```diff
@@ -113,7 +113,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 264,
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
This doesnt need to change since these changes do not actually touch any live runtime code. Benchmarking is entirely gated by a feature flag, and should never make its way onto the network.

Reply:
I want to agree, but the [CI failure](https://gitlab.parity.io/parity/substrate/-/jobs/810060) on https://github.com/paritytech/substrate/pull/8104/commits/24cf315123fb7dc8c6b01918a926738044e6c9a6 seems to indicate otherwise.

Reply:
CI isnt that smart. You should try to understand for yourself what `impl_version` means, and why this PR does not need to bump it. :)

I believe the CI merges just fine too without that test passing.

Reply:
OK, reverted that commit in e0444d8ae.